roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -100,13 +100,22 @@
     files:
     - "{{ tempdir }}/logging-elasticsearch-view-role.yaml"
     delete_after: true
-
 # configmap
+- fail:
+    msg: "The openshift_logging_es_log_appenders '{{openshift_logging_es_log_appenders}}' has an unrecognized option and only supports the following as a list: {{es_log_appenders | join(', ')}}"
+  when:
+    - es_logging_contents is undefined
+    - "{{ openshift_logging_es_log_appenders | list | difference(es_log_appenders) | length != 0 }}"
```

Comment:
I believe es_log_appenders is a list in vars/main.yaml.  This is how it was done in the original role, and I believe we added it as a way to ensure you defined atleast one appender